freight/api/base.py

```diff
@@ -32,7 +33,7 @@ def is_authorized(self):
         if method != 'Key':
             return False
 
-        if payload != current_app.config['API_KEY']:
+        if not compare_digest(payload.decode('utf8'), current_app.config['API_KEY'].decode('utf8')):
```

Comment:
This is technically unrelated, but I happened to notice it and we should be using a constant time comparison here.